Stripe/PublicHeaders/STPPaymentMethodCardChecks.h

```diff
@@ -25,19 +33,31 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)init __attribute__((unavailable("You cannot directly instantiate an STPPaymentMethodCardChecks.")));
 
 /**
- If a address line1 was provided, results of the check, one of ‘pass’, ‘failed’, ‘unavailable’ or ‘unchecked’.
+ If a address line1 was provided, results of the check.
  */
-@property (nonatomic, nullable, readonly) NSString *addressLine1Check;
+@property (nonatomic, readonly) STPPaymentMethodCardCheckResult addressLine1Check;
 
 /**
- If a address postal code was provided, results of the check, one of ‘pass’, ‘failed’, ‘unavailable’ or ‘unchecked’.
+ If a address postal code was provided, results of the check.
  */
-@property (nonatomic, nullable, readonly) NSString *addressPostalCodeCheck;
+@property (nonatomic, readonly) STPPaymentMethodCardCheckResult addressPostalCodeCheck;
 
 /**
- If a CVC was provided, results of the check, one of ‘pass’, ‘failed’, ‘unavailable’ or ‘unchecked’.
+ If a CVC was provided, results of the check.
  */
-@property (nonatomic, nullable, readonly) NSString *cvcCheck;
+@property (nonatomic, readonly) STPPaymentMethodCardCheckResult cvcCheck;
+
+/**
+@param string a string representing the check result as returned from
+the Stripe API
+ 
+@see https://stripe.com/docs/api/payment_methods/object#payment_method_object-card-checks
+
+@return an enum value mapped to that string. If the string is unrecognized or nil,
+returns STPPaymentMethodCardCheckResultUnknown.
+*/
++ (STPPaymentMethodCardCheckResult)checkResultFromString:(nullable NSString *)string;
```

Comment:
I don't think we need/want to make this part of the public API.

The only place I found where we're exposing our string/enum conversion function is `STPCard.h`, and that's marked as deprecated. The others are either only in the implementation, or are in a `+Private.h`